### PR TITLE
VBM-159: items screen layout

### PR DESCRIPTION
**Pull request by mazurGit**
https://user-images.githubusercontent.com/69375560/189757023-cce1413f-cf69-48b4-9f22-aaae79f320e0.mp4

### CODE REVIEW

File: packages/mobile/src/components/screen-wrapper/screen-wrapper.tsx [hunk 1]
```diff
@@ -1,16 +1,24 @@
...
+  statusBarProps?: StatusBarProps;
+  containerProps?: SafeAreaViewProps;
+};
+
+const ScreenWrapper: FC<Props> = ({
   style,
   children,
-  ...restProps
+  containerProps,
+  statusBarProps,
```
Comment (AleksandrTermenzhy) [inline packages/mobile/src/components/screen-wrapper/screen-wrapper.tsx:21]: Just apply new status bar component wherever you need it. No need to pass its props here

File: packages/mobile/src/navigation/screen-options/items-screen-options.tsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+type Props = {
+  route: RouteProp<RootNavigationParamList>;
+};
+
+const itemsScreenOptions = ({ route }: Props): NativeStackNavigationOptions => {
+  const getTitle = (screenName: RootScreenName) => {
+    switch (screenName) {
+      case RootScreenName.ITEMS_AND_SERVICES:
+        return t('items_and_services.TITLE');
```
Comment (AleksandrTermenzhy) [inline packages/mobile/src/navigation/screen-options/items-screen-options.tsx:18]: Single case `switch`?

File: packages/mobile/src/screens/items-and-services/items-and-services.tsx [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+const ItemsAndServices = () => {
+  const { dark, colors } = useCustomTheme();
+  const styles = useMemo(() => createStyles(colors), [dark, colors]);
+  const products = useAppSelector(selectProducts);
+
+  return (
+    <ScreenWrapper statusBarProps={styles.statusBar}>
+      <View style={globalStyles.px4}>
+        <View
```
Comment (AleksandrTermenzhy) [inline packages/mobile/src/screens/items-and-services/items-and-services.tsx:23]: Let's render applied filters as ListHeaderComponent

File: packages/mobile/src/navigation/screen-options/items-screen-options.tsx [hunk 4]
```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { NativeStackNavigationOptions } from '@react-navigation/native-stack';
+import { BurgerMenu, ButtonText } from '~/components/components';
+import { globalStyles } from '~/styles/styles';
+import { t } from 'i18next';
```
Comment (AleksandrTermenzhy) [inline packages/mobile/src/navigation/screen-options/items-screen-options.tsx:5]: Move it up pls

File: packages/mobile/src/navigation/screen-options/items-screen-options.tsx [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+import { RootNavigationParamList } from '~/common/types/types';
+import { RouteProp } from '@react-navigation/native';
```
Comment (AleksandrTermenzhy) [inline packages/mobile/src/navigation/screen-options/items-screen-options.tsx:7]: Also this one

File: packages/mobile/src/navigation/screen-options/items-screen-options.tsx [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+import { RootScreenName } from '~/common/enums/enums';
+
+type Props = {
+  route: RouteProp<RootNavigationParamList>;
+};
+
+const itemsScreenOptions = ({ route }: Props): NativeStackNavigationOptions => {
```
Comment (AleksandrTermenzhy) [inline packages/mobile/src/navigation/screen-options/items-screen-options.tsx:14]:
```suggestion
const getItemsScreenOptions = ({ route }: Props): NativeStackNavigationOptions => {
```

File: packages/mobile/src/screens/items-and-services/items-and-services.tsx [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+
+  return (
+    <ScreenWrapper>
+      <StatusBar
+        backgroundColor={colors.backgroundSecondary}
+        barStyle="dark-content"
+      />
+      <ListHeader contentContainerStyles={globalStyles.px4} />
+      <FlatList
+        style={globalStyles.px4}
```
Comment (AleksandrTermenzhy) [inline packages/mobile/src/screens/items-and-services/items-and-services.tsx:26]:
```suggestion
      <FlatList
        ListHeaderComponent={ListHeader}
        ListHeaderComponentStyle={globalStyles.px4}
        style={globalStyles.px4}
```

**Review by AleksandrTermenzhy — APPROVED** [review]

**Review by dolphin323 — APPROVED** [review]
Merge after fixing comments

**Review by VladimirBludov — APPROVED** [review]

**Review by seal323 — APPROVED** [review]

**Pull request merged**